File: app/src/main/java/com/example/samson/immvrse/SignUpActivity.java
package com.example.samson.immvrse;

import android.app.Activity;
import android.content.Intent;
import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;

import com.example.samson.immvrse.databinding.ActivitySignupBinding;

public class SignUpActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_signup);

        ActivitySignupBinding binding = DataBindingUtil.setContentView(this, R.layout.activity_signup);

        binding.signUp.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(SignUpActivity.this, OnBoardingActivity.class);
                startActivity(intent);
            }
        });

        binding.signIn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(SignUpActivity.this, LogInActivity.class);
                startActivity(intent);
            }
        });


    }
}

File: app/src/main/java/com/example/samson/immvrse/SignUpActivity.kt
package com.example.samson.immvrse

import android.content.Intent
import android.graphics.Typeface
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.text.method.PasswordTransformationMethod
import kotlinx.android.synthetic.main.activity_signup.*

class SignUpActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_signup)

        sign_up.setOnClickListener {
            startActivity(Intent(this, RegistrationSuccessActivity::class.java))
        }

        sign_in.setOnClickListener {
            startActivity(Intent(this, LogInActivity::class.java))
        }

        password.typeface = Typeface.DEFAULT
        password.transformationMethod = PasswordTransformationMethod()
    }
}

